ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -4623,6 +4623,47 @@ def test_paddle_tensor_unbind(
     )
 
 
+# uniform_
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="uniform_",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
```suggestion
        available_dtypes=helpers.get_dtypes("valid"),
```

This should always be valid- if the function only supports floats, then that's what valid will return.